spec/support/test_pals.rb

```diff
@@ -0,0 +1,252 @@
+      email: 'jodi@demo.studentinsights.org',
+      full_name: 'Teacher, Jodi',
+      school: @shs,
+      homeroom: @shs_jodi_homeroom
+    })
+
+    # Bill Nye is a biology teacher at Somerville High School.  He teaches sections
+    # on Tuesday and Thursday and has a homeroom period.
+    @shs_bill_nye_homeroom = FactoryGirl.create(:homeroom, {
+      name: 'SHS 917',
+      grade: '9',
+      school: @shs
+    })
+    @shs_bill_nye = FactoryGirl.create(:educator, {
+      email: 'bill@demo.studentinsights.org',
+      full_name: 'Teacher, Bill',
+      school: @shs,
+      homeroom: @shs_bill_nye_homeroom
+    })
+    @shs_biology_course = FactoryGirl.create(:course, school: @shs)
+    create_section_assignment(@shs_bill_nye, [
+      @shs_tuesday_biology_section = FactoryGirl.create(:section, course: @shs_biology_course),
+      @shs_thursday_biology_section = FactoryGirl.create(:section, course: @shs_biology_course)
+    ])
+
+    # Hugo teachers two sections of ceramics at the high school.
+    @shs_hugo_art_teacher = FactoryGirl.create(:educator, {
+      email: "hugo@demo.studentinsights.org",
+      full_name: 'Teacher, Hugo',
+      local_id: '650',
+      school: @shs
+    })
+    @shs_ceramics_course = Course.create(course_number: "ART-302", course_description: "Ceramic Art 3", school: @shs)
+    create_section_assignment(@shs_hugo_art_teacher, [
+      @shs_second_period_ceramics = Section.create(section_number: "ART-302A", term_local_id: "FY", schedule: "2(M,R)", room_number: "201", course: @shs_ceramics_course),
+      @shs_fourth_period_ceramics = Section.create(section_number: "ART-302B", term_local_id: "FY", schedule: "4(M,R)", room_number: "234", course: @shs_ceramics_course)
+    ])
+
+    # Fatima teaches two sections of physics at the high school.
+    @shs_fatima_science_teacher = FactoryGirl.create(:educator, {
+      email: "fatima@demo.studentinsights.org",
+      full_name: 'Teacher, Fatima',
+      local_id: '750',
+      school: @shs
+    })
+    @shs_physics_course = Course.create(course_number: "SCI-201", course_description: "Physics 2", school: @shs)
+    create_section_assignment(@shs_fatima_science_teacher, [
+      @shs_third_period_physics = Section.create(section_number: "SCI-201A", term_local_id: "S1", schedule: "3(M,W,F)", room_number: "306W", course: @shs_physics_course),
+      @shs_fifth_period_physics = Section.create(section_number: "SCI-201B", term_local_id: "S1", schedule: "5(M,W,F)", room_number: "306W", course: @shs_physics_course)
+    ])
+
+    # Mari is a freshman at the high school, enrolled in biology and in Jodi's homeroom.
+    @shs_freshman_mari = FactoryGirl.create(:student, {
+      first_name: 'Mari',
+      last_name: 'Kenobi',
+      school: @shs,
+      homeroom: @shs_jodi_homeroom,
+      grade: '9'
+    })
+    FactoryGirl.create(:student_section_assignment, student: @shs_freshman_mari, section: @shs_tuesday_biology_section)
+
+    self
+  end
+
+  def index!
```

Comment:
This came up in the `sign_in` test, that it was relying on `student_risk_level` and that wasn't being created here, so this adds it.